cEpiabm/src/sweeps/household_sweep.cpp

```diff
@@ -81,7 +81,7 @@ namespace epiabm
 
         double infectiousness = Covidsim::CalcHouseInf(infector, timestep);
         double susceptibility = Covidsim::CalcHouseSusc(infector, infectee, timestep);
-        double foi = infectiousness * susceptibility;
+        double foi = infectiousness * susceptibility * 0.5;
```

Comment:
Think you have accidentally merged in an old version of the C++ - @NicholasFan235 changed this the other way recently? Maybe worth checking no other changes have been reversed?

Reply:
so we DON'T want the factor of 0.5?

Reply:
No, but I think it must have been added by merging back in an old branch so worth checking with @NicholasFan235  whether any other changes have been overwritten too